lms/djangoapps/teams/serializers.py

```diff
@@ -0,0 +1,140 @@
+"""
+Defines serializers used by the Teams API.
+"""
+
+from django.contrib.auth.models import User
+from rest_framework import serializers
+from openedx.core.lib.api.fields import ExpandableField
+from .models import CourseTeam, CourseTeamMembership
+from openedx.core.djangoapps.user_api.serializers import UserSerializer
+
+
+class CollapsedUserSerializer(serializers.HyperlinkedModelSerializer):
+    """
+    Serializes users in a collapsed format, with just a username and url.
+    """
+    url = serializers.HyperlinkedIdentityField(view_name='accounts_api', lookup_field='username')
+
+    class Meta(object):
+        """
+        Defines meta information for the ModelSerializer.
+        """
+        model = User
+        fields = ("username", "url")
+        read_only_fields = ("username",)
+
+
+class CollapsedCourseTeamSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
Is it possible to have a general purpose 'collapsed reference serializer' that includes an id and a URL. I would like to get to the point where every API supports expanded and collapsed versions of its references, and it would be preferable if new serializers were not needed for every new reference.